### PR TITLE
Nerfs Goof for the sake of everyone's sanity.

### PR DESCRIPTION
**Pull request by deathride58**
Someone has to do it.
:cl: deathride58
balance: The webhook will now tell Iamgoofball to stop every time he makes a new PR.
/:cl:

### CODE REVIEW

**Comment by JJRcop** [comment]
Ey 🍻

**Comment by Buggy123** [comment]
A well reasoned, modest, practical and useful PR. :+1:

**Comment by Iamgoofball** [comment]
8===d lok

File: tools/WebhookProcessor/github_webhook_processor.php [hunk 1]
```diff
@@ -370,6 +370,8 @@ function handle_pr($payload) {
 				$author = $payload['pull_request']['user']['login'];
 				if(isset($balances[$author]) && $balances[$author] < 0 && !is_maintainer($payload, $author))
 					create_comment($payload, 'You currently have a negative Fix/Feature pull request delta of ' . $balances[$author] . '. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.');
+				else if($author == 'Iamgoofball')
+					create_comment($payload, 'Stop. Please. Just stop for the sake of everyones sanity');
```
Comment (QualityVan) [inline tools/WebhookProcessor/github_webhook_processor.php]: everyone's

**Comment by The-clown-known-as-Pepe** [comment]
Go back to your downstream you illegal alien

🤠 yeehaw

**Comment by lzimann** [comment]
Whatever

**Comment by ACCount12** [comment]
nooooooo

**Comment by deathride58** [comment]
>15 thumbs up
>no downvotes
>closed before 24 hour mark

lzimann is an unashamed goof apologist and should feel ashamed

**Pull request closed without merging**